code-challenges.js

```diff
@@ -15,35 +15,128 @@ const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
 
+
+
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+// Pseudo:
+// - create a function called boilingPoint
+// - input - temp
+// - use conditional statement to make an evaluation on the temperature
+// - if temp is at or below 42, output temp is below boiling point
+// - if temp is at or above 212, output temp is at boiling point
+// - if temp is at or above 350, output temp is above boiling point
+
+
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
 
+const boilingPoint = (temp) => {
+  if(temp <= temp1){
+  return `${temp} is below boiling point`
+} else if (temp >= temp2){
+  return `${temp} is above boiling point`
+} else if (temp >= temp3){
+  return `${temp} is at boiling point`
+} else if(temp > temp1){
+  return `${temp} is close to boiling point`
+}else {
+  return ""
+  }
+}
+console.log(boilingPoint(41))
+console.log(boilingPoint(42))
+console.log(boilingPoint(100))
+console.log(boilingPoint(212))
+console.log(boilingPoint(250))
+console.log(boilingPoint(350))
+console.log(boilingPoint(351))
```

Comment:
I like that you are checking multiple other numbers in the function, you can also pass the variables defined above your function into the argument when you invoke the function.  It is great that you added an else to cover any edge cases, it is helpful to add something into the string, even if it is saying "sorry, incorrect input"